src/ui/server/shared-validation/phone-number/index.ts

```diff
@@ -1,9 +1,32 @@
+import Joi from 'joi';
 import { PhoneNumberUtil } from 'google-libphonenumber';
 import generateValidationErrors from '../../helpers/validation';
 import { DEFAULT_COUNTRY_UK } from '../../constants';
 
 const phoneUtil = PhoneNumberUtil.getInstance();
 
+const pattern = '^[0-9-)(+— ]*$';
+
+/**
+ * validates phone number is valid and returns a boolean
+ * @param {string} phoneNumber
+ * @returns {Boolean}
+ */
+const phoneNumberValidity = (phoneNumber: string) => phoneUtil.isValidNumberForRegion(phoneUtil.parse(phoneNumber, DEFAULT_COUNTRY_UK), DEFAULT_COUNTRY_UK);
+
+/**
+ * joi phone using regex to ensure only numbers and specified special characters are in the string
+ * @param {string} phoneNumber
+ * @returns {Boolean}
+ */
+const phoneNumberValidation = (phoneNumber: string) => {
+  const schema = Joi.string().regex(RegExp(pattern)).required();
+
+  const validation = schema.validate(phoneNumber);
+
+  return validation.error;
```

Comment:
Does validation always return `.error`?

Reply:
Just thinking seems we need to add an if statement because this could error if `.error` does not exist?